src/content/en/2021/css.md

```diff
@@ -17,9 +17,22 @@ featured_stat_3: TODO
 featured_stat_label_3: TODO
 ---
```

Comment:
@meyerweb could you also start filling in the metadata at the top of the file?

Reply:
Yep, will do.